app/models/concerns/export/exporter.rb

```diff
@@ -23,10 +28,8 @@ def setup_export
         include_deleted: @include_deleted,
         version: @version,
       }
-      options[:export_id] = Digest::MD5.hexdigest(options.to_s)[0..31]
-
-      @export = GrdaWarehouse::HmisExport.create(options)
-      @export.fake_data = GrdaWarehouse::FakeData.where(environment: @faked_environment).first_or_create
+      @options[:export_id] = Digest::MD5.hexdigest(@options.to_s)[0..31]
```

Comment:
Should this contain a timestamp, or are we deliberately generating the same ExportID for the same configuration?

Reply:
```suggestion
      @options[:export_id] ||= Digest::MD5.hexdigest(@options.to_s)[0..31]
```
Or do we need to recalculate this because more data can be added to the `@options` hash?